client/components/timezone-dropdown/docs/example.jsx

```diff
@@ -34,7 +34,7 @@ export default React.createClass( {
 					<a href="/devdocs/design/timezone-dropdown">TimezoneDropdown</a>
 				</h2>
 
-				<Card style={ { width: '300px', margin: 0 } }>
+				<Card style={ { width: '300px', height: '350px', margin: 0 } }>
```

Comment:
I don't think this is specific to this instance of the dropdown component, but it seems they do not expand beyond the `id="content"` node of the page. I think that's likely not a common issue, but one we should be careful about. I don't think we need to address it in this PR though.
![screen shot 2016-03-23 at 4 27 04 pm](https://cloud.githubusercontent.com/assets/134044/14001506/325da272-f114-11e5-8949-6462e0eaa43c.png)